infra/tf/main.tf

```diff
@@ -96,13 +112,23 @@ resource "digitalocean_firewall" "ursa-network" {
     source_addresses = ["0.0.0.0/0", "::/0"]
   }
   inbound_rule {
-    port_range = "4070"
-    protocol   = "tcp"
+    protocol         = "tcp"
+    port_range       = "3000"
     source_addresses = ["0.0.0.0/0", "::/0"]
   }
   inbound_rule {
-    port_range = "6009"
-    protocol   = "tcp"
+    port_range       = "4069"
+    protocol         = "tcp"
+    source_addresses = ["0.0.0.0/0", "::/0"]
+  }
+  inbound_rule {
+    port_range       = "6009"
+    protocol         = "tcp"
+    source_addresses = ["0.0.0.0/0", "::/0"]
+  }
+  inbound_rule {
+    port_range       = "4890"
+    protocol         = "udp"
     source_addresses = ["0.0.0.0/0", "::/0"]
   }
```

Comment:
6009 and 80/443 should only be exposed. In this case only 6009 for ursa network

Reply:
What about our public quic port 4890 for libp2p? Enabled by default on all ursa nodes

And 3000 would be nice to be able to get to the grafana dashboard to watch the node health

Will remove 4069 though had it on for testing

Reply:
yeah, 4069 shouldn't be exposed,  4890 yeah that could stay, everything else is a balance. I suppose a metrics port is fine